frontend/app/configuration/Config.scala

```diff
@@ -49,14 +49,9 @@ object Config {
   def idWebAppSigninUrl(uri: String): String =
     (idWebAppUrl / "signin") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
 
-  def idWebAppRegisterUrl(uri: String, abTestVariant: CheckoutFlowVariant = A): String ={
-
-    val idMember = "clientId" -> abTestVariant.identitySkin
-
-    (idWebAppUrl / "register") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & idMember
-  }
-
-
+  def idWebAppRegisterUrl(uri: String, abTestVariant: CheckoutFlowVariant = A): String =
+    (idWebAppUrl / "register") ? ("returnUrl" -> s"$membershipUrl$uri") & idSkipConfirmation & "clientId" -> abTestVariant.identitySkin
```

Comment:
I'm not sure what the evaluation precedence is on this, so just check you stil get the right value, or surround `"clientId" -> abTestVariant.identitySkin` with parentheses!